tsconfig.json

```diff
@@ -12,7 +12,8 @@
         "sourceMap": true,
         "typeRoots": ["./node_modules/@0x/typescript-typings/types", "./node_modules/@types"],
         "noUnusedParameters": true,
-        "resolveJsonModule": true
+        "resolveJsonModule": true,
+        "allowSyntheticDefaultImports": true
```

Comment:
This is needed for the `apollo-client` which `0x/mesh-graphql-client` uses under the hood